pkg/cmd/init.go

```diff
@@ -181,7 +181,12 @@ func certSetup(cfg *config.Config) (*certchains.CertificateChains, error) {
 						Name:         "kubelet-server",
 						ValidityDays: cryptomaterial.ShortLivedCertificateValidityDays,
 					},
-					Hostnames: []string{cfg.Node.HostnameOverride, cfg.Node.NodeIP},
+					// Kubelet hostnames must contain those addresses advertised by a node. In MicroShift these
+					// correspond to the node's IP address and the hostname. In order to accommodate having more
+					// than one node for CNCF certification tests, the subject alternative names from the
```

Comment:
Let's not mention the tests here. We may have other reasons to deal with multi-node in the future.

```suggestion
					// than one node, the subject alternative names from the
```

Reply:
Done